binding.cc

```diff
@@ -278,21 +289,32 @@ namespace zmq {
     int value = Nan::To<int32_t>(info[1]).FromJust();
 
     Context *context = GetContext(info);
-    if (zmq_ctx_set(context->context_, option, value) < 0)
-      return Nan::ThrowError(ExceptionFromError());
+    while (zmq_ctx_set(context->context_, option, value))
+      if (zmq_errno() != EINTR)
+        return Nan::ThrowError(ExceptionFromError());
     return;
   }
 
-  NAN_METHOD(Context::GetOpt) {
+  NAN_METHOD(Context::Get) {
     if (info.Length() != 1)
       return Nan::ThrowError("Must pass an option");
     if (!info[0]->IsNumber())
       return Nan::ThrowTypeError("Option must be an integer");
     int option = Nan::To<int32_t>(info[0]).FromJust();
 
     Context *context = GetContext(info);
-    int value = zmq_ctx_get(context->context_, option);
-    info.GetReturnValue().Set(Nan::New<Integer>(value));
+
+    int rc;
+    do {
+      rc = zmq_ctx_get(context->context_, option);
+      if (rc < 0) {
+        if (zmq_errno() != EINTR)
+          return Nan::ThrowError(ExceptionFromError());
+        continue;
+      }
+    } while (rc < 0);
```

Comment:
is `rc` a flag to indicate success or failure, or does it return a meaningful option value?